### PR TITLE
Theme folder reorg

### PR DESCRIPTION
**Pull request by jbrinley**
A partial reorganization of the core plugin's `Theme` namespace.

### CODE REVIEW

File: docs/frontend/components/card.md [hunk 1]
```diff
@@ -111,13 +111,7 @@ And the panel controller using the Card component
...
 namespace Tribe\Project\Templates\Controllers\Panels;
 
-use Tribe\Project\Panels\Types\Cardgrid as CardGridPanel;
-use Tribe\Project\Templates\Components\Button;
-use Tribe\Project\Templates\Components\Card;
-use Tribe\Project\Templates\Components\Image;
-use Tribe\Project\Templates\Components\Text;
-use Tribe\Project\Templates\Components\Title;
-use Tribe\Project\Theme\Image_Sizes;
+use Tribe\Project\Panels\Types\Cardgrid as CardGridPanel;use Tribe\Project\Templates\Components\Button;use Tribe\Project\Templates\Components\Card;use Tribe\Project\Templates\Components\Image;use Tribe\Project\Templates\Components\Text;use Tribe\Project\Templates\Components\Title;use Tribe\Project\Theme\Config\Image_Sizes;
```
Comment (dpellenwood) [inline docs/frontend/components/card.md:114]: Not sure what happened to formatting here.

File: wp-content/plugins/core/src/Theme/Config/Web_Fonts.php [hunk 2]
```diff
@@ -0,0 +1,134 @@
+<?php
```
Comment (dpellenwood) [inline wp-content/plugins/core/src/Theme/Config/Web_Fonts.php:1]: I think @faction23 is in the process of removing our webfont loader JS-based implementation here. We'll just have to be aware of merge conflicts when we get there.

File: wp-content/themes/core/template-tags/titles.php [hunk 3]
```diff
@@ -8,31 +8,38 @@
 /**
  * Sets up the proper title
  *
+ * @todo  Remove this function
```
Comment (dpellenwood) [inline wp-content/themes/core/template-tags/titles.php:11]: :100: I've got on my list to eliminate all the template-tag functions this go-round.

File: wp-content/plugins/core/src/Theme/Config/Web_Fonts.php (hunk 2 above)
Comment (faction23) [reply wp-content/plugins/core/src/Theme/Config/Web_Fonts.php:1]: Yes i am doing a fair amount of work on these theme files right now

File: wp-content/themes/core/template-tags/titles.php (hunk 3 above)
Comment (faction23) [reply wp-content/themes/core/template-tags/titles.php:11]: nice

**Pull request merged**